src/app/pages/ecommerce/crazydomains.css

```diff
@@ -0,0 +1,18 @@
+/* TODO: Move this to ecommerce module */
+:root {
+  --nfd-ecommerce-accent-primary: #196bde;
+  --nfd-ecommerce-accent-secondary: #84c0fc;
+  --nfd-ecommerce-accent-tertiary: #006dc7;
+  --nfd-ecommerce-border-primary: #dbdbdb;
+  --nfd-ecommerce-border-secondary: #dce2ea;
+  --nfd-ecommerce-bg-primary: #e5f6ff;
+  --nfd-ecommerce-bg-secondary: #eff6fc;
+  --nfd-ecommerce-bg-tertiary: #f5f5f5;
+  --nfd-ecommerce-text-primary: #333333;
+  --nfd-ecommerce-text-secondary: #425466;
+  --nfd-ecommerce-text-tertiary: #5b5b5b;
+  --nfd-ecommerce-font-primary: -apple-system, BlinkMacSystemFont, "Segoe UI",
+    Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  --nfd-ecommerce-font-secondary: -apple-system, BlinkMacSystemFont, "Segoe UI",
+    Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
Shouldn't these ecommerce specific variables should just inherit the values of more global variables unless they explicitly need to be overridden?

We set the primary variables here: https://github.com/newfold-labs/wp-plugin-crazy-domains/blob/main/src/app/stylesheet.scss#L1-L21

Also, I haven't built this PR to test it yet, but the hex codes here don't seem to match up with the other branded colors I just linked above. Not sure which are correct for CD's styles, but just want to make sure we are consistent.

Reply:
The colors are a WIP, waiting on designs to update these variables and yes you're right, they'll inherit from the plugin.

Reply:
Agreed that these should inherit from the app/stylesheet.scss, if there are any colors missing, let's add them at the app level and then use the variables in the module styles.